jtreg builds are broken

https://ci.adoptopenjdk.net/view/Dependencies/job/dependency_pipeline/ seems broken for a while now. Specifically, JDK 20 head will need `JTreg 7+1` at least and it would be good to get it building. See

https://mail.openjdk.org/pipermail/jdk-dev/2022-September/006906.html

The build fails with:
```
***********************************************
Building JTREG jtregtip...
***********************************************
JAVA_HOME: /usr/lib/jvm/java-1.8.0-openjdk
Changing into /home/adoptopenjdk/workspace/dependency_pipeline/jtreg
Currently in /home/adoptopenjdk/workspace/dependency_pipeline/jtreg
Removing contents of build folder
[build.sh][INFO] CYGWIN_OR_MSYS=0
[build.sh][ERROR] JDK 11 or newer is required to build jtreg
```

The build machine will have to have JDK 11+ and possibly JDK 8 installed going forward.

--- a/tools/code-tools/jtreg.sh
+++ b/tools/code-tools/jtreg.sh
@@ -6,6 +6,7 @@ set -eu
 readonly JTREG_5='jtreg5.1-b01'
 readonly JTREG_6='jtreg-6+1'
 readonly JTREG_6_1='jtreg-6.1+1'
+readonly JTREG_7='jtreg-7+1'
 
 function checkWorkspaceVar()
 {
@@ -45,12 +46,19 @@ buildJTReg()
     elif [ "$1" == "$JTREG_6_1" ]; then
       export JTREG_BUILD_NUMBER="1"
       export BUILD_VERSION="6.1"
+    elif [ "$1" == "$JTREG_7" ]; then
+      export JTREG_BUILD_NUMBER="1"
+      export BUILD_VERSION="7"
+      export JAVA_HOME=/usr/lib/jvm/jdk-11.0.16.1+1
+      export PATH=$PATH:$JAVA_HOME/bin
     fi
     git checkout $version
   else
     unset BUILD_NUMBER
     unset BUILD_VERSION
     unset JTREG_BUILD_NUMBER
+    export JAVA_HOME=/usr/lib/jvm/jdk-11.0.16.1+1
+    export PATH=$PATH:$JAVA_HOME/bin
     git checkout master
   fi
 
@@ -110,5 +118,7 @@ cd "$WORKSPACE"
 buildJTReg "$JTREG_5"
 buildJTReg "$JTREG_6"
 buildJTReg "$JTREG_6_1"
+buildJTReg "$JTREG_6_1"
+buildJTReg "$JTREG_7"
 buildJTReg
 echo '...finished with build process.'

